toolkit/scripts/toolkit.mk

```diff
@@ -128,17 +128,32 @@ $(valid_arch_spec_names): $(go-specarchchecker) $(chroot_worker) $(local_specs)
 		--log-file="$(valid_arch_spec_names_logs_path)" \
 		--log-color="$(LOG_COLOR)"
 
-##help:target:install-prereqs=Install build prerequisites automatically.
+##help:target:install-prereqs=Install basic build prerequisites automatically.
```

Comment:
We also want to VERY clearly call out what people should do when they see the go version check fail (tools.mk I think?).

We should have a very clear error that says "Use `make install-prereqs-and-configure` to avoid this in the future" or somthing.